kong/runloop/handler.lua

```diff
@@ -1113,11 +1113,6 @@ return {
     before = function(ctx)
       local server_port = var.server_port
       ctx.host_port = HOST_PORTS[server_port] or server_port
-
```

Comment:
removed as it seems not being read afterwards

Reply:
that was because of a bug